### PR TITLE
Add a very basic Image component

### PR DESCRIPTION
**Pull request by corranwebster**
...because using a Chaco `ImagePlot` to display a simple `.png` is Wrong.

### CODE REVIEW

**Comment by jwiggins** [comment]
@itziakos: Do you have time to finish this before the end of the month? I'd like to make it part of the next release.

File: examples/enable/image_draw.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+THIS_DIR = os.path.split(__file__)[0]
+
+class MyFrame(DemoFrame):
+
+    def _create_window(self):
+        path = os.path.join(THIS_DIR, 'deepfield.jpg')
+        image = Image.from_file(path)
+        image.resist_width = 'weak'
+        image.resist_height = 'weak'
```
Comment (corranwebster) [inline examples/enable/image_draw.py]: +1 on that.  I'll quickly add that to the PR.

**Comment by corranwebster** [comment]
I'm writing up some tests now.

**Comment by corranwebster** [comment]
@jwiggins The tests are pretty comprehensive at this point.  I think that this is good to be merged.

**Comment by jwiggins** [comment]
Sweet! Thanks for the fast turnaround @corranwebster

**Comment by jwiggins** [comment]
Thanks for the feature! :+1:

[Coverage Status](https://coveralls.io/builds/1698506)

**Pull request merged**